requirements.txt

```diff
@@ -0,0 +1,2 @@
+adafruit-ws2801 >= 1.0.1
+flask >= 1.1.1
```

Comment:
Good initiative! I think an installation / requirements section should actually go in the README.md file.
A quick check revealed that https://github.com/adafruit/Adafruit_Python_WS2801/ is outdated, replaced by https://github.com/adafruit/Adafruit_CircuitPython_WS2801.

Reply:
requirements.txt is the way to go. See: https://pip.readthedocs.io/en/1.1/requirements.html
Seems deprecated indeed